locksmith/__tests__/operations/wedlocksOperations.test.ts

```diff
@@ -102,14 +104,20 @@ describe('Wedlocks operations', () => {
         },
         network
       )
-      const transferUrl = `${[
+
+      const baseAppUrl =
         process.env.UNLOCK_ENV !== 'prod'
           ? 'https://staging-app.unlock-protocol.com'
-          : 'https://app.unlock-protocol.com',
-      ]}/transfer?lockAddress=0x95de5F777A3e283bFf0c47374998E10D8A2183C7&keyId=&network=${network}`
+          : 'https://app.unlock-protocol.com'
```

Comment:
Should move this to config I think. This variable can be re-used in a few places.